### PR TITLE
Ref:implementation conversion lua table  to ObjC Dictionary

### PR DESCRIPTION
**Pull request by yuxiaoen**
添加一个实现，从lua的table -> 转化成ObjC的字典的一个方法

### CODE REVIEW

**Comment by yuxiaoen** [comment]
i found some param is wrong,then i'm fixed and restart commit

**Review by yuxiaoen — COMMENTED** [review]
please rebuild?

**Comment by yuxiaoen** [comment]
rebuilt?

File: cocos/scripting/lua-bindings/manual/platform/ios/CCLuaObjcBridge.mm [hunk 1]
```diff
@@ -14,6 +14,41 @@
     lua_setglobal(L, "LuaObjcBridge");
 }
 
+
+static void luaTableWithObjcDictionary(lua_State *L, NSMutableDictionary *dict,NSString *key){
```
Comment (minggo) [inline cocos/scripting/lua-bindings/manual/platform/ios/CCLuaObjcBridge.mm]: how about change function name to `luaTableToObjcDictionary`

**Comment by minggo** [comment]
and could you please add a test case?

**Comment by minggo** [comment]
thanks, will you please add a test case?

**Comment by yuxiaoen** [comment]
ok,write in the project?

**Comment by minggo** [comment]
You can add it into lua-tests

**Comment by yuxiaoen** [comment]
i add it in LuaBridgeTest.lua, does it look alright?

**Comment by minggo** [comment]
Thanks, it looks great. I will test it ASAP.

**Comment by minggo** [comment]
When tested on Mac, met this error log:

```
[LUA-print] luaoc.callStaticMethod("LuaObjectCBridgeTest", "printLuaTableToDictionary", "table: 0x0cb86b00") - error: [-5] INVALID METHOD SIGNATURE
```

**Comment by yuxiaoen** [comment]
sorry,it's my mistake,i submit the file in "mac" directory

**Pull request merged**